lib/mongoose_push.ex

```diff
@@ -13,6 +13,8 @@ defmodule MongoosePush do
   require Logger
   alias MongoosePush.Service
 
+  @behaviour MongoosePushBehaviour
```

Comment:
This module does not conform to this behaviour yet. And if it does, then behaviour definition is incorrect (should expect correct structs that we pass from controller as input, while this module does not support it yet).

Reply:
done